Sources/UXKit/AppKit/UXTableView-AppKit.swift

```diff
@@ -61,8 +74,27 @@
     static var automatic = slideDown
   }
 
-  public extension NSTableView {
+public extension UXTableView  {
```

Comment:
_Within_ UXKit I think it is preferable to use the native names in extensions, as it makes it obvious what the extension is specifically for.